packages/web-app-external/src/index.js

```diff
@@ -0,0 +1,40 @@
+import translations from '../l10n/translations'
+
+import App from './App.vue'
+import store from './store'
+
+const appInfo = {
+  name: 'External',
+  id: 'external'
+}
+
+const routes = [
+  {
+    name: 'apps',
+    path: '/:app/:file_id',
+    components: {
+      app: App
+    }
+  }
+]
+
+async function fetchAvailableMimeTypes() {
+  const vueStore = window.Vue.$store
+  if (!vueStore.getters.capabilities.files.app_providers[0].enabled) {
```

Comment:
```suggestion
  if (!vueStore.getters.capabilities.files.app_providers[0]?.enabled) {
```
Please use optional chaining here. Otherwise there will be an error if the `app_providers` capability is empty.